libs/androidlib/src/mill/androidlib/AndroidModule.scala

```diff
@@ -118,6 +120,14 @@ trait AndroidModule extends JavaModule {
     super.repositoriesTask() :+ AndroidSdkModule.mavenGoogle
   }
 
+  override def checkGradleModules: T[Boolean] = true
+  override def resolutionParams: Task[ResolutionParams] = Task.Anon {
+    super.resolutionParams().addVariantAttributes(
+      "org.jetbrains.kotlin.platform.type" ->
+        VariantMatcher.AnyOf(Seq(VariantMatcher.Equals("androidJvm"), VariantMatcher.Equals("jvm")))
```

Comment:
ok this is what I meant in https://github.com/com-lihaoyi/mill/pull/4626#issuecomment-2827984709, that looks fine with me!